build_tools/aws-sdk-code-generator/lib/aws-sdk-code-generator/views/endpoints_module.rb

```diff
@@ -42,10 +42,6 @@ def initialize(options)
 
         # @return [Array<EndpointParameter>]
         attr_reader :parameters
-
-        def has_endpoint_built_in?
```

Comment:
Why is this being removed?

Reply:
I have this in the account id change that was reviewed by us .. but I decided to carry it forward here - it's really just a syntax change. Is it incorrect? I think Endpoint is ALWAYS present in rules is it not? In the switch case below, it is switching on built in, and on SDK::Endpoint, we do the same check that was previously in the mustache template - check regional endpoint and either nil or endpoint.

Reply:
I don't think it MUST be present in the rules, but the code we have for constructing the SDK::Endpoint parameter is already conditional, so I agree this is fine, I just wanted to check.

Reply:
Even if Endpoint isn't guaranteed to exist, I think it's still functionally equivalent, since the switch case looks for Endpoint anyway.